treeherder/log_parser/tasks.py

```diff
@@ -1,11 +1,16 @@
+import json
 import logging
 
 import newrelic.agent
 from celery.exceptions import SoftTimeLimitExceeded
+from requests import HTTPError
```

Comment:
Let's keep the import's original *explicit* form:
```python
from requests.exceptions import HTTPError
```

Reply:
Solved.